src/cpp/rtps/participant/RTPSParticipantImpl.h

```diff
@@ -412,7 +412,7 @@ class RTPSParticipantImpl
 
     WLP* wlp();
 
-    fastdds::dds::builtin::TypeLookupManager* typelookup_manager() const;
+    fastdds::dds::builtin::TypeLookupManager& typelookup_manager() const;
```

Comment:
Does the RTPSParticipantImpl need to define this accesor? Or was it something to expose the TypeLookup manager publicly?

Reply:
This is needed for private access